packages/prosemirror-lwdita/src/github-integration/request.ts

```diff
@@ -167,6 +165,12 @@ export function processRequest(): undefined | URLParams {
     try {
       const parameters = getAndValidateParameterValues(currentUrl);
 
+      // Check if the "welcome" note has not yet been dismissed
+      // and show it on page load
+      if (!hasConfirmedNotification()) {
```

Comment:
This function call seems to be in inappropriate place as most of `processRequest` code does this 
```ts
/**
 * Process the URL parameters and handle the notifications
 */
```
showing the welcome dialog in here does not fit the description of the function.

Reply:
@marmoure I don't understand your comment: This function is solely checking a certain key in the localStorage which creates the condition for showing the note. This is the only place where the application handles request and routing, so I see this is the only place to call this function.

Reply:
> so I see this is the only place to call this function.
I think it's more appropriate if this code is placed in `example.ts` before we make the call to `processRequest`

Reply:
Fixed with 1f1b3bb